index.html

```diff
@@ -167,7 +173,7 @@ <h2 class="works-item-title"> Tonic </h2>
                 </div>
             </div>
             <div class="works-item works-item-4">
-                <div class="works-item-image-4"></div>
+                <div class="works-item-image-4 works-item-image"></div>
                 <div class="works-item-info-block">
                     <div class="works-item-project-info">
```

Comment:
- [ ] Kindly space your contents in all your working cards to match the Figma design. The image and the description are very close

![image](https://user-images.githubusercontent.com/98453979/193312568-1d2c537f-bd0d-46e0-a0cd-7b46d653a9a1.png)

![image](https://user-images.githubusercontent.com/98453979/193312680-517c7e95-8d41-4234-838b-a445d85cc3f5.png)

### Checkout Figma

![image](https://user-images.githubusercontent.com/98453979/193312782-429a424b-c4cb-4b19-a84c-0ee5e83a272b.png)

- [ ] Kindly align the contents of your `about me` section to the center of the page to match the Figma design.

![image](https://user-images.githubusercontent.com/98453979/193313283-6d997f7d-4ff2-4030-b07e-903c5f7560d6.png)

- [ ] Kindly note that there are two lines separating frameworks and skills.
![image](https://user-images.githubusercontent.com/98453979/193313856-28312e70-f85d-486a-894b-c38177a21448.png)

### Checkout figma

![image](https://user-images.githubusercontent.com/98453979/193314118-e8463917-51fa-4964-ad4d-4ffcacddf199.png)

- [ ] Kindly align the text in your `get in touch` button to the center to match the Figma design

![image](https://user-images.githubusercontent.com/98453979/193314412-1bbd6915-2baf-4a34-b178-67c4d0dad31f.png)

### Checkout Figma

![image](https://user-images.githubusercontent.com/98453979/193314539-3bb10c5a-4926-499f-8e36-b0fd46a2b5a0.png)